src/components/BaseComponent.tsx

```diff
@@ -11,7 +11,7 @@ const BaseComponent: React.FC<BaseComponentProps> = ({
 }) => {
   return (
     <div
-      className={`bg-primary h-screen w-screen p-8 grid grid-cols-6 gap-4 ${baseComponentStyles}`}
+      className={`bg-primary w-screen p-8 grid grid-cols-6 gap-4 ${baseComponentStyles}`}
```

Comment:
What is the purpose of this change?

Reply:
it was creating fixed height for background

Reply:
what do you mean?

Reply:
revert this change